libs/shared/config-cat-core/src/lib/context-transformer.ts

```diff
@@ -49,11 +49,11 @@ function stringOrUndefined(param?: unknown): string | undefined {
   return undefined;
 }
 
-export function transformContext(context: EvaluationContext): ConfigCatUser | never {
+export function transformContext(context: EvaluationContext): ConfigCatUser | undefined {
```

Comment:
The `transformContextValues` function should be revised as newer versions of ConfigCat SDKs [don't require string values for custom user attribute values](https://github.com/configcat/common-js/blob/v9.3.0/src/User.ts#L12-L51) any more.

Custom attribute values should be passed to the SDK as is to let it deal with them internally. But at least, `Date` objects shouldn't be converted to ISO strings because that won't work with ConfigCat's BEFORE/AFTER date comparators.

Reply:
Oh did not see that, will also update that! :)